engine/src/execution_graph/logic_controllers/BatchAggregationProcessing.h

```diff
@@ -131,13 +149,20 @@ class ComputeAggregateKernel : public kernel {
 private:
     std::vector<AggregateKind> aggregation_types;
     std::vector<int> group_column_indices;
+    std::vector<std::string> aggregation_input_expressions;
+    std::vector<std::string> aggregation_column_assigned_aliases;
 };
 
 class DistributeAggregateKernel : public kernel {
 public:
 	DistributeAggregateKernel(std::size_t kernel_id, const std::string & queryString, std::shared_ptr<Context> context, std::shared_ptr<ral::cache::graph> query_graph)
 		: kernel{kernel_id, queryString, context, kernel_type::DistributeAggregateKernel} {
         this->query_graph = query_graph;
+        std::tie(this->group_column_indices, this->aggregation_input_expressions, this->aggregation_types,
```

Comment:
why did this change?

Reply:
This just moved this code from the run() function tot he constructor